### PR TITLE
Add Web Components Badges

### PR DESCRIPTION
**Pull request by jshcrowthe**
This adds an apprentice, journeyman, and master level of the web components badges. 

I will be committing more to this PR over the next 2-3 days and should have it complete by end of week. If you have feedback please leave it inline and I will address it as it comes.

### CODE REVIEW

**Comment by spacerockzero** [comment]
oooh nice. i will look at this tomorrow!

**Comment by jshcrowthe** [comment]
This is good to go, please review and if good to go I'll move to `badges-active`

File: badges-in-development/web-components/_micro_POJ_webcomponents.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+-----
+
+## Intro
+
+> “In an ideal scenario, the HTML language would be expressive enough to create complex UI widgets and also extensible so that we, the developers, could fill in any gaps with our own tags. Today, this is finally possible through a new set of standards called Web Components.” ― https://css-tricks.com/modular-future-web-components/
+
+-----
+
+## Suggested Assignment(s)
+
```
Comment (spacerockzero) [inline badges-in-development/web-components/_micro_POJ_webcomponents.md]: This should say "Choose one:", so they know they only need to complete one main item.

Comment (jshcrowthe) [inline badges-in-development/web-components/_micro_POJ_webcomponents.md]: Added that, woops :)

File: badges-in-development/web-components/_micro_vulzanice.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
+# Web Components - Vulcanize [Micro Badge]
```
Comment (spacerockzero) [inline badges-in-development/web-components/_micro_vulzanice.md]: file name typo ^ ^

File: badges-in-development/web-components/_micro_vulzanice.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+-----
+
+## Assignment(s)
+
+Choose one:
+
+1) Make a creative project that demonstrates the following:
+- [ ] Vulcanization of a group of elements
+- [ ] Inlining of external scripts/styles
+- [ ] Integration of vulcanziation into a build pipeline
```
Comment (spacerockzero) [inline badges-in-development/web-components/_micro_vulzanice.md]: sppelling

File: badges-in-development/web-components/_micro_web_component_perf.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
+# Web Components - Vulcanize [Micro Badge]
```
Comment (spacerockzero) [inline badges-in-development/web-components/_micro_web_component_perf.md]: stale name ^^

**Comment by spacerockzero** [comment]
This looks pretty good. Do you know when you might have the time estimates for it? (you can merge without, but they are super-helpful for users, so we should get them done ASAP)

**Comment by spacerockzero** [comment]
Soon after this gets merged, I will update the README and some other stuff so we can bring these to the active group right away.

**Pull request merged**